vertx/src/main/java/module-info.java

```diff
@@ -17,6 +18,7 @@
             com.microsoft.gctoolkit.api;
 
     provides com.microsoft.gctoolkit.jvm.JavaVirtualMachine with
-            PreUnifiedJavaVirtualMachine;
+            PreUnifiedJavaVirtualMachine,
```

Comment:
@leveretconey - Would you mind getting rid of the import statements and using the fully qualified class names in the provides statement? Using fully qualified class names rather than imports is the accepted style for module-info files.